airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/UsagePerConnectionTable.tsx

```diff
@@ -142,7 +142,11 @@ const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ credi
         accessor: "creditsConsumed",
         collapse: true,
         customPadding: { right: 0 },
-        Cell: ({ cell }: CellProps<FullTableProps>) => <UsageValue>{cell.value}</UsageValue>,
+        Cell: ({ cell }: CellProps<FullTableProps>) => (
+          <UsageValue>
+            <FormattedNumber value={cell.value} />
```

Comment:
Did you consider whether to use `minimumFractionDigits`/`maximumFractionDigits` to always show a set number of decimal points? Not sure I have a strong opinion, just curious if that might look better for consistency (e.g. `0.00` instead of `0`)

Reply:
Agree.  I was also on the fence about it but the existing usage (credits remaining) didn't do that.  I'll swap them all out!